src/workerd/api/node/buffer.c++

```diff
@@ -201,10 +206,11 @@ uint32_t writeInto(
           str.size());
     }
     case Encoding::HEX: {
-      KJ_STACK_ARRAY(kj::byte, buf, string->Length(), 1024, 536870888);
-      string->WriteOneByte(js.v8Isolate, buf.begin(), 0, -1,
-                           v8::String::NO_NULL_TERMINATION |
-                           v8::String::REPLACE_INVALID_UTF8);
+      KJ_STACK_ARRAY(kj::byte, buf, string.length(js), 1024, 536870888);
```

Comment:
The size of the maxStack value, 512MB - 24, stood out when reading the PR, it is much larger than a typical stack. For ew/workerd, it's okay because kj doesn't use the value when building with clang.